app/assets/stylesheets/layout.scss

```diff
@@ -68,11 +57,6 @@ main {
   fill: currentColor;
 }
 
-.site-header {
```

Comment:
@brandonrosage Did you want to set the colors in the HTML of the header partial itself? Since this was removed the header on some pages is not black anymore